tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/SearchConfigSpec.scala

```diff
@@ -0,0 +1,596 @@
+package ch.epfl.bluebrain.nexus.tests.kg
+
+import akka.http.scaladsl.model.StatusCodes
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.tests.BaseSpec
+import ch.epfl.bluebrain.nexus.tests.Identity.resources.Rick
+import ch.epfl.bluebrain.nexus.tests.iam.types.Permission.{Organizations, Resources}
+import io.circe.Json
+import monix.bio.Task
+import org.scalatest.Assertion
+
+class SearchConfigSpec extends BaseSpec {
```

Comment:
It looks ok for now but we may want to break it further down  at some point.
In other tests, we may also want to run some aggregations too (again, not in this PR)